README.md

```diff
@@ -1,9 +1,39 @@
 Cloudinary Media Editing SDK
-=========================
+============================
+
 ## About
 The Cloudinary Media Editing SDK allows you to quickly and easily integrate your application with Cloudinary.
 Effortlessly edit and deliver your cloud's assets.
 
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Media Editing SDK Guide](https://cloudinary.com/documentation/media_editing_api).
+
+
+## Table of Contents
+- [Key Features](#key-features)
```

Comment:
please add to your table of contents all the other features listed in your examples

Reply:
Done